src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -8233,6 +8407,7 @@ export class StudyViewPageStore
     });
 
     readonly initialMolecularProfileSampleCounts = remoteData({
+        await: () => [this.samples, this.molecularProfiles],
```

Comment:
curious why this becomes necessary.  we need to be careful about adding load dependencies as they can affect performance of the study view pretty drastically